test/tests/p5.Gain.js

```diff
@@ -0,0 +1,51 @@
+const expect = chai.expect;
+let gain;
+
+describe('p5.Gain', function () {
+  beforeEach(function () {
+    gain = new p5.Gain();
+  });
+
+  it('can be created', function () {
+    expect(gain.input).to.have.property('gain');
+    expect(gain.output).to.have.property('gain');
+    let audioContext = gain.ac;
+    expect(audioContext).to.have.property('baseLatency').to.be.an('number');
+    expect(audioContext).to.have.property('destination');
+    expect(audioContext).to.have.property('state').to.be.an('string');
+  });
+  it('can be created and disposed', function () {
+    gain.dispose();
+    expect(gain).to.not.have.property('input');
+    expect(gain).to.not.have.property('output');
+  });
+
+  describe('methods', function () {
+    describe('setInput', function () {
+      it('can set Input', function () {
+        let soundFile = p5.prototype.loadSound('./testAudio/drum.mp3');
+        gain.setInput(soundFile);
+      });
+    });
+    describe('connect, disconnect', function () {
+      it('can connect to empty node', function () {
```

Comment:
```suggestion
      it('connects to p5.soundOut when no arguments are provided', function () {
```

just observing that this is what happens when no arguments are provided

https://github.com/processing/p5.js-sound/blob/17987e714cf5bce6904e31912603a0d640e8d0c9/src/gain.js#L106-L107

Reply:
Yep, looks good